arrows/ocv/register_algorithms.cxx

```diff
@@ -71,84 +70,288 @@ namespace kwiver {
 namespace arrows {
 namespace ocv {
 
-/// Register OCV algorithm implementations with the given or global registrar
-int register_algorithms( vital::registrar &reg )
+extern "C"
+KWIVER_ALGO_OCV_EXPORT
+void
+register_factories( kwiver::vital::plugin_loader& vpm )
 {
+  static auto const module_name = std::string( "algorithms.ocv" );
+  if (vpm.is_module_loaded( module_name ) )
+  {
+    return;
+  }
+
 #if defined(HAVE_OPENCV_NONFREE)
   cv::initModule_nonfree();
 #endif
 
-  REGISTRATION_INIT( reg );
+  // add factory               implementation-name       type-to-create
+  auto fact = vpm.ADD_ALGORITHM( "ocv", kwiver::arrows::ocv::analyze_tracks );
+  fact->add_attribute( kwiver::vital::plugin_factory::PLUGIN_DESCRIPTION,
+                    "" )
+    .add_attribute( kwiver::vital::plugin_factory::PLUGIN_MODULE_NAME, module_name )
+    .add_attribute( kwiver::vital::plugin_factory::PLUGIN_VERSION, "1.0" )
+    .add_attribute( kwiver::vital::plugin_factory::PLUGIN_ORGANIZATION, "Kitware Inc." )
+    ;
+
+
+  fact = vpm.ADD_ALGORITHM( "ocv", kwiver::arrows::ocv::draw_tracks );
+  fact->add_attribute( kwiver::vital::plugin_factory::PLUGIN_DESCRIPTION,
+                    "" )
+    .add_attribute( kwiver::vital::plugin_factory::PLUGIN_MODULE_NAME, module_name )
+    .add_attribute( kwiver::vital::plugin_factory::PLUGIN_VERSION, "1.0" )
+    .add_attribute( kwiver::vital::plugin_factory::PLUGIN_ORGANIZATION, "Kitware Inc." )
+    ;
+
+
+  fact = vpm.ADD_ALGORITHM( "ocv", kwiver::arrows::ocv::estimate_fundamental_matrix );
+  fact->add_attribute( kwiver::vital::plugin_factory::PLUGIN_DESCRIPTION,
+                    "" )
```

Comment:
Use this

    Use OpenCV to estimate a fundimental matrix from feature matches